clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -8192,6 +8193,9 @@ public void testListClientMetricsResourcesNotSupported() {
     @Test
     public void testCallFailWithUnsupportedVersionExceptionDoesNotHaveConcurrentModificationException() throws InterruptedException {
         try (AdminClientUnitTestEnv env = new AdminClientUnitTestEnv(mockCluster(1, 0))) {
+            // mock AdminMetadataManager for KafkaAdminClient, so closing the default one
```

Comment:
It seems you don't use the `env` actually. Maybe you can remove the env from this test case and create `MockClient` by yourself.

Reply:
Yes, updated it. Thanks for the review.